OpenEdXMobile/src/main/java/org/edx/mobile/view/WebViewDiscoverFragment.java

```diff
@@ -76,6 +77,19 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         }
     }
 
+    private void initTitle() {
+        Bundle args = getArguments();
+        if (args != null) {
+            String screenTitle = args.getString(Router.EXTRA_SCREEN_TITLE);
```

Comment:
Same here.
Why we are fetching screen_title?

Reply:
We are showing the toolbar based on this title. Title is only shown for Post Login experience,